### PR TITLE
add migration schemas to ruler docker image

### PR DESCRIPTION
**Pull request by jtlisi**
The ruler can currently interact with a postgres db directly without a configdb. However, it does not have the migration files. This PR adds the migration files to the ruler image.

If anyone has an idea of a place to put these files so they can live in the docker build context of the configdb, ruler, and lite images let me know. I wasn't sure the best way to do that without having the make process copy them there.
Signed-off-by: Jacob Lisi <jacob.t.lisi@gmail.com>

### CODE REVIEW

**Comment by tomwilkie** [comment]
> I wasn't sure the best way to do that without having the make process copy them there.

This I think.  Can you add some makefile magic to copy them in please?

**Comment by jtlisi** [comment]
@tomwilkie 
This should be good for a second look. I squashed it into a single commit.

**Comment by tomwilkie** [comment]
LGTM; might want to tweak it so you don't have to mount the migrations into the root of the build container, but not too fussed.

@bboreham @gouthamve need a second pair of eyes.

File: Makefile [hunk 1]
```diff
@@ -98,7 +102,7 @@ configs-integration-test: build-image/$(UPTODATE)
 		-v $(shell pwd)/.cache:/go/cache \
 		-v $(shell pwd)/.pkg:/go/pkg \
 		-v $(shell pwd):/go/src/github.com/cortexproject/cortex \
-		-v $(shell pwd)/cmd/configs/migrations:/migrations \
+		-v $(shell pwd)/migrations:/migrations \
```
Comment (jtlisi) [inline Makefile]: @tomwilkie this change is actually only in the in the integration test container. The build container does not mount the migrations

Comment (tomwilkie) [reply Makefile]: +1

File: Makefile [hunk 2]
```diff
@@ -34,13 +34,17 @@ images:
 PROTO_DEFS := $(shell find . $(DONT_FIND) -type f -name '*.proto' -print)
 PROTO_GOS := $(patsubst %.proto,%.pb.go,$(PROTO_DEFS))
 
+# Ensure a each image that requires a migration directory mounted has the updated migrations directory in their
```
Comment (bboreham) [inline Makefile]: It took me a while to work out what was going on.  Maybe point to the `cp -r` line lower down which does the actual work?

**Comment by jtlisi** [comment]
@bboreham Changed as requested, let me know if there is anything else

**Pull request merged**